nostr-client-plus/src/bin/aggregator.rs

```diff
@@ -453,9 +453,9 @@ async fn finalize_classification(
                 break;
             }
             let result_identifier = task.get_result_identifier();
-            tracing::info!("result_identifier: {}", result_identifier.to_string());
-            if !result_identifier.is_empty() {
-                match answers.entry(result_identifier) {
+            // tracing::info!("result_identifier: {}", result_identifier.to_string());
```

Comment:
let's just delete this instead of comment it out. we can always add it back later